form-sharing/3-updateSheetsList.js

```diff
@@ -0,0 +1,89 @@
+fn(state => {
+  const koboFormsData = state.filteredKoboFormsData;
```

Comment:
same here, you don't have to reassign `state.filteredKoboFormsData`, 
You can pluck it out from state and use it without reassignment 
```
const { filteredKoboFormsData } = state;

state.sheetsData = filteredKoboFormsData.map((form) => {
  return [
    form.uid,
    form.name,
    createTagName(form.name),
    form.owner__username,
    createInstance(form.name),
    createProjectId(form.name),
    createGRMiD(form.name),
    createWorkspaceName(form.name),
    form.url,
    form.date_modified,
    form.date_created,
  ];
});
```